samples/helloworld-ts/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
When we are correctly supporting erasable syntax, the tsconfig.json file will be entirely unnecessary.

Reply:
We should remove this file now.